apollo-compatibility/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM hseeberger/scala-sbt:17.0.2_1.9.9_3.1.1 AS build
```

Comment:
Stupid question, but any reason that we can't use the same one used by CI? I believe it has `sbt` and all that installed as well

Reply:
TBH I just copied this almost wholesale from https://github.com/sangria-graphql/sangria-federated/pull/287